README.md

```diff
@@ -26,43 +27,44 @@ Steel flexi-daggers with a rubber blunt, or a nylon trainer. Additional control
 Rapier Fencing Equipment
 
 Armour:
-1. Helmet (12-kilogram three-weapon/sabre fencing mask or better) with back of head protection.
+1. Helmet (12-kilogram three-weapon/sabre fencing mask or better). No cuts to the back of the head are allowed unless the helmet includes back of the head protection.
 2. Gorget
-3. Leather gauntlets with a high cuff
+3. Leather fencing gloves or gauntlets. The wrist must be protected by the gauntlet cuff or the jacket.
```

Comment:
Kept leather material requirement because it is the standard for soft fencing gloves.